server/internal/adapter/gql/resolver_mutation_field.go

```diff
@@ -35,7 +43,7 @@ func (r *mutationResolver) CreateField(ctx context.Context, input gqlmodel.Creat
 	}
```

Comment:
line 22~43 should be done in the usecase layer. CreateField usecase should accept modelID and metadata bool. The adapter layer should be thin as possible.

Reply:
do you mean line 22~39?  lines 40~43 are using a gqlmodel convertor and we are following the same logic everywhere, If so we need to move this logic from the update function as well